### PR TITLE
add conflict handling for existing files

### PR DESCRIPTION
**Pull request by scramb**
## Description

Add error handling for file upload. 
If file exists on Client and Server decline upload. Open notification with error message.

## Related Issue

- Refs #476

## How Has This Been Tested?

- :hand: 

## Types of changes

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] Code changes

### CODE REVIEW

File: apps/files/src/components/FileUpload.vue [hunk 1]
```diff
@@ -34,18 +35,27 @@ export default {
...
       let files = e.target.files || e.dataTransfer.files
       if (!files.length) return
       for (let file of files) {
-        this.upload(file)
+        let exists = find(this.files, ['name', file.name])
+        if(!exists){
+          this.upload(file)
+        } else {
+          this.showNotification({
+            title: `Upload for ${file.name} failed - File allready exists`,
```
Comment (DeepDiver1975) [inline apps/files/src/components/FileUpload.vue]: We need to handle Translations here

File: apps/files/src/components/FileUpload.vue [hunk 2]
```diff
@@ -52,7 +52,7 @@ export default {
           this.upload(file)
         } else {
           this.showNotification({
-            title: `Upload for ${file.name} failed - File allready exists`,
+            title: `${this.$gettext('Upload for')} ${file.name} ${this.$gettext('failed - File allready exists')}`,
             type: 'error'
```
Comment (DeepDiver1975) [inline apps/files/src/components/FileUpload.vue]: Interpolation required. Position of the variable in the string is different depending on the language

**Comment by scramb** [comment]
little problem with interpolation this is not supported by easygettext which we use to extract. I'll take a look if i can maybe get this upstream a bit faster

**Comment by DeepDiver1975** [comment]
Rebasing right now .... will merge right away ...

**Pull request merged**